From 0d12f1c69aa754c7fa9f5ab58756b19f1d8e06ec Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Thu, 7 Mar 2013 15:32:49 +0200
Subject: [PATCH] stap-prep: debian: do not check for linux-kbuild

linux-headers* already depends on the right linux-kbuild* package and
our heuristic for guessing the package name is lacking. For example
with linux-image-3.2.0-0.bpo.4-amd64 version 3.2.35-2~bpo60+1 from
squeeze-backports it predicts "linux-kbuild-3.2.0" while the package
is actually named "linux-kbuild-3.2".
---
 stap-prep |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/stap-prep b/stap-prep
index 4a4c87c..541316a 100755
--- a/stap-prep
+++ b/stap-prep
@@ -77,7 +77,6 @@ esac
     echo "make >= 0"
     echo "linux-image-$ABINAME = $VERSION"
     echo "linux-headers-$ABINAME = $VERSION"
-    echo "linux-kbuild-$BASEVERSION >= 0"
     case "$DISTRO" in
 	Debian)
 	    echo "linux-image-$ABINAME-dbg = $VERSION"
-- 
1.7.2.5

